src/services/workflow.service.js

```diff
@@ -19,32 +19,13 @@ import { GQuery } from '@/services/gquery'
 import store from '@/store/'
 import Alert from '@/model/Alert.model'
 import { createApolloClient } from '@/utils/graphql'
-import gql from 'graphql-tag'
-
-// TODO: remove these once the api-on-the-fly mutations are hooked into the UI
-const HOLD_WORKFLOW = gql`
-mutation HoldWorkflowMutation($workflow: WorkflowID!) {
-  hold (workflows: [$workflow]) {
-    result
-  }
-}
-`
-
-const RELEASE_WORKFLOW = gql`
-mutation ReleaseWorkflowMutation($workflow: WorkflowID!) {
-  release (workflows: [$workflow]){
-    result
-  }
-}
-`
-
-const STOP_WORKFLOW = gql`
-mutation StopWorkflowMutation($workflow: WorkflowID!) {
-  stop (workflows: [$workflow]) {
-    result
-  }
-}
-`
```

Comment:
Moved it out to the `graphql/query` where we have other query strings or `DocumentNode`s.